networking/network_observability/json-flows-format-reference.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
```

Comment:
should we generate directly `network-observability-flows-format.adoc` as an ASSEMBLY file, so that we only have 1 file for it? This would avoid having a first title `JSON flows format reference` and then a sub-title `Network Flows format specifications` which looks redundant with the top title

Reply:
Yeah let's actually do that vs having these two files.

Reply:
Sorry actually, just Slacked you about this but I think I have to leave it as two files. But for your work, I think its unnecessary to create two files. I have the contents of this file copied over to my PR exactly, only one sentence is in the Assembly file and the rest is in the Reference module. The API reference is similarly documented now that I look back at the PR history, you created one file and I split it into two files with just one sentence copied over to the Assembly.